src/liteclient.h

```diff
@@ -26,7 +26,7 @@ class Lock {
 
 class LiteClient {
  public:
-  LiteClient(Config& config_in, const std::shared_ptr<AppEngine>& app_engine = nullptr);
+  LiteClient(Config& config_in, const std::shared_ptr<AppEngine>& app_engine = nullptr, bool finalize = true);
```

Comment:
maybe set it to false by default? seems less dangerous

Reply:
just wondering, instead of a flag, why dont we add it to the config class?

Reply:
It was always "true" before the flag was introduced, hence I decided to leave it as a default.

It will require making changes to libaktualizr while we would like to reduce dependency on it.

Reply:
and if instead of a flag we create an Actions class? not sure if in the future we will need to perform some other actions?

Reply:
also as a general rule, would it be possible if we use smaller names?
ie finalizeInstallation --> completeInstall or  doInstall? 
I find it much easier to read and remember (but this is just a personal perference)

Reply:
this is the name of the package manager interface, so it makes sense to name in the way the underlying code uses, so we have consistent terminology.

Reply:
>if instead of a flag we create an Actions class?

it looks like overengineering in the given context. I would rather consider refactoring an overall main.cc with something as you are proposing, but I would do it only after we have a decent CI/CD  in place to make sure we don't introduce any regressions.